src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_ThemeResource.cs

```diff
@@ -70,100 +64,87 @@ public async Task When_Parent_Resource_Override_On_Loaded()
 #if __MACOS__
 		[Ignore("Currently fails on macOS, part of #9282 epic")]
 #endif
-		public async Task When_DefaultForeground_Non_Fluent() => await When_DefaultForeground(Colors.Black, Colors.White);
+		public async Task When_DefaultForeground_Fluent() => await When_DefaultForeground(Colors.Black, Colors.White);
```

Comment:
This change (and the below one) don't look right. It's swapping the test expectation for fluent and non-fluent.